gym/envs/tests/test_envs_semantics.py

```diff
@@ -0,0 +1,85 @@
+import numpy as np
+import json
+import hashlib
+import os
+
+import logging
+logger = logging.getLogger(__name__)
+
+import gym
+from gym import envs, spaces
+
+from generate_json import create_rollout
+
+DATA_DIR = os.path.dirname(__file__)
+ROLLOUT_FILE = os.path.join(DATA_DIR, 'rollout.json')
+ROLLOUT_STEPS = 100
+episodes = ROLLOUT_STEPS
+steps = ROLLOUT_STEPS
+
+if not os.path.isfile(ROLLOUT_FILE): 
+  with open(ROLLOUT_FILE, "w") as outfile:
+    json.dump({}, outfile, indent=2)
+
+def test_env_semantics(spec):
+  with open(ROLLOUT_FILE) as data_file:
+    rollout_dict = json.load(data_file)
+
+  if spec.id not in rollout_dict:
+    added = create_rollout(spec)
```

Comment:
I see, I was doing this for the workflow of generating a rollout for a new environment if a rollout doesn't already exist and the env shouldn't be skipped by this test. Alternatively, I can add a function to generate_json to generate the new rollouts for new environments when explicitly called. Does that sound like a better solution?